cypress/tests/ab-testing.cy.ts

```diff
@@ -0,0 +1,38 @@
+describe('A/B Testing Page', () => {
+  const PAGE_URL = 'abtest';
+
+  describe('Navigation', () => {
+    it('Should be linked correctly from home page', () => {
+      // Visit home page and click the correct link
+      cy.visit('/');
+      cy.get('a').contains('A/B Testing').click();
+
+      // Assert correct page loads
+      cy.url().should('contain', PAGE_URL);
+    });
+  });
+
+  describe('Functionality', () => {
+    beforeEach(() => {
+      // Directly navigate to a/b testing page
+      cy.visit(PAGE_URL);
+    });
+
+    it('Should successfully display page contents', () => {
+      const HEADER_TEXT = 'A/B Test';
```

Comment:
These two const declarations could be moved outside of the it(), to keep the code within the block strictly focused on test steps

Reply:
Done